main.cpp

```diff
@@ -1,9 +1,13 @@
 #include <SFML/Graphics.hpp>
 #include "Game.hpp"
+#include <iostream>
+#include "RandomGenerator.hpp"
+
 
 int main()
 {
     Game game;
     game.run();
+
```

Comment:
Nadal są tu zmiany: linijka 5 i 11.

Do zrobienia:
- [ ] jeszcze raz przywrócić pierwotne zmiany pliku main.cpp

Reply:
@JakubMastalski jeśli chcesz robić jakiekolwiek poprawki w plikach to nie w tym PR. Każdy PR ma swoje przeznaczenie. Ten jest od generatora, więc nie wprowadzaj poprawek nie związanych z tym tematem w tym PR i na tej gałęzi. To podstawowa zasada.

Reply:
zastanawia mnie jezeli tak jak teraz zrobilem to zle i zmienilem cos na galezi randomGenrator w mainie to develop i master sa za tym i jezeli chce zmienic teraz do domyslnego to musze to dokona cpoprawek na randomGenerator
bo na tamtych galeziach jest wczesniejsza wersja nie wiem czy dobrze wytlumaczylem ewetualnie powiem ci na zajeciach

Reply:
@KodKarls